### ISSUE TITLE
[UI] Support upgrading RKE2 in Rancher Dashborad

### ISSUE THREAD

**Issue by davidnuzik** [rancher/rancher#28247]
Depends on RKE2 ("Rancher cluster") backend work as per https://github.com/rancher/rke2/issues/155

Once Menna has this ready for UI, the frontend will need to tie into this and support upgrading clusters.

**Comment by davidnuzik** [comment]
Closing. The RKE2 upgrades in Rancher Dashboard are no longer a GA requirement. We might revisit this later. If so, a new issue will be created later with details.
